app/misc/easemob.py

```diff
@@ -0,0 +1,95 @@
+# coding=utf-8
+
+import requests
+import json
+from requests.auth import AuthBase
+
+
+class Easemob(object):
+    """SDK for Easemob IM system"""
+    JSON_HEADER = {'content-type': 'application/json'}
+
+    def __init__(self, app):
+        self.app_id = app.config['EASEMOB_APP_ID']
```

Comment:
It means getting value of `EASEMOB_APP_ID` from the config of the parameter `app`, and it does works.